tests/manual/test-procedure.rst

```diff
@@ -446,6 +446,25 @@ Test Case: Autocomplete
 - [ ] Disable autocomplete in "Options" and confirm that autocomplete no longer offers suggestions.
 
 
+Test Case A11y: Keyboard focus order follows the visual layout
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+- [ ] Using your keyboard, navigate through the menu and any submenus using the `tab` and `enter` or `space` keys
+and confirm  they follow natural order of the page elements eg.left to right and top top bottom.
+- [ ] Using your keyboard, navigate through the modal windows using the `tab` and `enter` or `space` keys
+and confirm  they follow natural order of the page elements eg.left to right and top top bottom.
```

Comment:
I know we haven't been very good at this in the past, but since we are breaking these entries into multiple lines, could we keep lines at 80 characters or less?